tests/unit/modal_test.coffee

```diff
@@ -74,3 +74,16 @@ test 'Test preserving the focus when clicking on non-focusable element', ->
   click(modalBody)
   .then ->
     validateFocus buttonCancel, 'The focus should stay on the Cancel button'
+
+test 'Test pressing Enter to confirm', ->
+  expect 1
+
+  modal = @subject
+    enterToConfirm: yes
+  spy = sinon.spy modal, "send"
+
+  @append()
+  modalComponent = modal.$()
+  keyEvent(modalComponent, 'keydown', 13)
+  andThen ->
+    ok spy.calledOnce, "sendConfirm gets called when hitting enter"
```

Comment:
Can you use `calledWithArguments` or whatever it's called to make this more specific?